xlml/utils/xpk.py

```diff
@@ -58,13 +59,15 @@ def run_workload(
 
   cmds = (
       "set -x",
-      f"gcloud config set project {cluster_project}",
-      f"gcloud config set compute/zone {zone}",
       "git clone https://github.com/google/xpk.git /tmp/xpk",
       "cd /tmp/xpk",
-      "apt-get update && apt-get install -y google-cloud-sdk",
+      "curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | gpg --dearmor -o /usr/share/keyrings/cloud.google.gpg",
+      'echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list',
```

Comment:
This is great! Now that we're installing gcloud cli in the command, can we decouple from the workload image here? Try chaning [L84](http://shortn/_Ema6NScBxX) to `image='python:3.10'`.

Reply:
Good idea! I changed it and tested that it works.